.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,6 +14,12 @@
 <!-- We must be able to understand the design of your change from this description. If we can't get a good idea of what the code will be doing from the description here, the pull request may be closed at the maintainers' discretion. -->
 <!-- Keep in mind that the maintainer reviewing this PR may not be familiar with or have worked with the code here recently, so please walk us through the concepts. -->
 
+### Issue
+
+<!-- Put in the note about what issue is resolved by this PR, especially if it is a GitHub issue. It should be in the form of "Resolves #N", where N is the number of the issue. More information about this is available in GitHub documentation: https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword -->
```

Comment:
Not only ```Resolves```. There is a list of commands.

Reply:
@iceseer yes, makes sense to add a few like "close" but when you give too many options it gets messy  (if you're curious https://journals.sagepub.com/doi/10.1111/j.1467-9280.2007.01906.x) :D 

Too many options is bad and no options is bad too - I'm gonna add a few more 👍